src/aznfswatchdog

```diff
@@ -133,24 +133,29 @@ check_for_regional_account()
 # we delete the entry so that kernel looks up fresh NAT rules and creates a new entry.
 #
 declare -A cthash_synsent
-declare -A cthash_synsent_now
 
 reconcile_conntrack_synsent()
 {
     local l_ip=$1
     local l_sport=$2
     local l_dport=$3
     local l_nfsip=$4
+    local seconds_remaining=$5
 
     key="${l_ip}:${l_sport}:${l_dport}:${l_nfsip}"
-    let cthash_synsent[$key]++
-    let cthash_synsent_now[$key]++
+    
+    if [ ! -v cthash_synsent[$key] ]; then
+        cthash_synsent[$key]=$seconds_remaining
+        return
+    fi
+
+    age_seconds=$(expr ${cthash_synsent[$key]} - $seconds_remaining)
 
     #
     # We are called every 5 secs, so this deletes an entry stuck in
     # SYN_SENT for 25/30 secs.
     #
-    if [ ${cthash_synsent[$key]} -ge 5 ]; then
+    if [ $age_seconds -ge 25 ]; then
         cmd="conntrack -D -p tcp -d $l_ip -r $l_nfsip --sport $l_sport --dport $l_dport"
         wecho "Deleting conntrack entry stuck in SYN_SENT state [$cmd]"
```

Comment:
"Deleting conntrack entry stuck in SYN_SENT state for $age_seconds seconds [$cmd]"

Reply:
Just a confirmation, age_seconds is the time >25, from the first detection by our code (not exact time). since we check every 5 seconds, so I also feel, it's mostly okay to print that. At max 5 seconds of difference bw exact and caught duration.